com/laboon/ToStringTest.java

```diff
@@ -0,0 +1,105 @@
+package com.laboon;
+
+import static org.junit.Assert.*;
+import org.junit.Test;
+import java.util.Random;
+
+public class ToStringTest {
+
+	World w;
+	Random r;
+
+	/*If a world of size zero is created, the toString() method
+	 * should return a string of size 3, because it will not execute any
+	 * of its for loops. Thus, the only characters in the string returned
+	 * from the toString() method will be two (2) spaces and one (1) newline
+	 * character.
+	 */
+	@Test
+	public void testZeroLengthWorld() {
+		w = new World(0, 2, 50);
+		
+		String testString = w.toString();
+		
+		assertEquals(testString.length(), 3);
+	}
+	
+	/*Two worlds constructed with identical conditions should return
+	 * strings of the same length upon invocation of their respective
+	 * toString() methods. World size was limited to 200 maximum
+	 * so as not to destroy my PC with a 999999 by 999999 world.
+	 */
+	@Test
+	public void testStringLength() {
+		
+		r = new Random();
```

Comment:
If you use a test double for Random, or set the size, seed, and percent yourself, you can reliably re-create worlds exactly as before.  Would be useful for checking exact output instead of properties.